minishift/README.adoc

```diff
@@ -0,0 +1,206 @@
+[[running-auth-cluster-services-on-openshift]]
+= Running Auth, Cluster services on OpenShift
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 2
+
+toc::[]
+
+These instructions will help you run your services on OpenShift using MiniShift.
+[[prerequisites]]
+== Prerequisites
+
+link:https://docs.openshift.org/latest/minishift/getting-started/installing.html[MiniShift v1.27.0]
+
+link:https://docs.okd.io/latest/cli_reference/get_started_cli.html=installing-the-cli[oc 3.11.0]
+
+link:https://www.linux-kvm.org/page/Downloads[KVM Hypervisor]
+
+[[install-minishift]]
+==== Install Minishift
+
+Make sure you have all prerequisites installed. Please check the list link:https://docs.openshift.org/latest/minishift/getting-started/installing.html=install-prerequisites[here]
+
+Download and put `minishift` in your $PATH by following steps link:https://docs.openshift.org/latest/minishift/getting-started/installing.html=manually[here]
+
+Verify installation by running following command, you should get version number.
+```bash
+minishift version
+```
+
+[[install-oc]]
+==== Install oc
+Please install and set up oc on your machine by visiting link:https://docs.openshift.org/latest/cli_reference/get_started_cli.html=installing-the-cli[oc]
+
+Verify installation by running following command, you should get version number.
+```bash
+oc version
+```
+
+[[usage]]
+== Usage
+
+When you want to run `fabric8-cluster` and it's dependent services i.e. `fabric8-auth` on OpenShift use following command:
+```
+make dev-openshift
+```
+Please enter sudo password when prompted, it is needed in order to create an entry in the `/etc/hosts`.
+`minishift ip` gives the IP address on which MiniShift is running. This automation creates a host entry as `minishift.local` for that IP. This domain is whitelisted on fabric8-auth.
```

Comment:
This domain is whitelisted on fabric8-**cluster**.

Reply:
no, it should `fabric8-auth` because we have whitelisted it in keycloak for auth not for cluster